components/features/Dashboard/Applications/ResponseTest/ResponseTest.tsx

```diff
@@ -1,9 +1,84 @@
-import React from 'react'
```

Comment:
Hello, Rashad! Out of curiosity, are all the import statements used in this file?

Reply:
Hi jo, oh yeah thanks for pointing that out, I've recently moved the inbox component to a separate component, I think the EmailTile import here is unnecessary.